integration-cli/docker_utils.go

```diff
@@ -205,7 +205,15 @@ func (d *Daemon) getClientConfig() (*clientConfig, error) {
 
 // Start will start the daemon and return once it is ready to receive requests.
 // You can specify additional daemon flags.
-func (d *Daemon) Start(arg ...string) error {
+func (d *Daemon) Start(args ...string) error {
+	logFile, err := os.OpenFile(filepath.Join(d.folder, "docker.log"), os.O_RDWR|os.O_CREATE|os.O_APPEND, 0600)
+	d.c.Assert(err, check.IsNil, check.Commentf("[%s] Could not create %s/docker.log", d.id, d.folder))
+
+	return d.StartWithTTY(logFile, args...)
+}
+
+// StartWithTTY will start the daemon and attach its streams to a given TTY.
+func (d *Daemon) StartWithTTY(tty *os.File, providedArgs ...string) error {
```

Comment:
Given this is really just sending the output to a File, not necessarily a tty, perhaps calling it `StartWithRedirect` instead?